2018/1/solution.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+def frequency_total(frequencies):
+    return sum([int(frequency) for frequency in frequencies])
```

Comment:
Nice use of a comprehension.  If you ever don't want a list comprehension, you could use a generator expression as well (sum(int(frequency) for frequency in frequencies).  Just in case you want to slim down the square brackets.